Makefile

```diff
@@ -1,13 +1,14 @@
 VERSION = 2
-LIBRARY_NAME = pam_watchid.so
+LIBRARY_PREFIX = pam_watchid
+LIBRARY_NAME = $(LIBRARY_PREFIX).so
 DESTINATION = /usr/local/lib/pam
-TARGET = x86_64-apple-macosx10.15
+TARGET = apple-darwin20.1.0
```

Comment:
It may have to resolve the target value from the system:
```
TARGET = $(swift -version)
```

Is this enough to work across different chip sets?